docker-compose.yaml

```diff
@@ -0,0 +1,21 @@
+version: '3.7'
+
+services:
+  db:
+    image: "postgres"
+    environment:
+      - "POSTGRES_HOST_AUTH_METHOD=trust"
+    volumes:
+      - postgres_data:/var/lib/postgresql/data/
+  web:
+    build: .
+    command: python /code/manage.py runserver 0.0.0.0:8000
+    volumes:
```

Comment:
If there is no important writes to the local filesystem. There isn't really a need for a volume mount.

Reply:
we decided to keep the volumes for now.